base/expr.jl

```diff
@@ -999,6 +999,12 @@ function findmeta_block(exargs, argsmatch=args->true)
 end
 
 remove_linenums!(ex) = ex
+
+"""
+    Base.remove_linenums!(ex)
+
+Remove all line-number metadata from expression-like object `ex`.
+"""
 function remove_linenums!(ex::Expr)
```

Comment:
```suggestion
"""
    Base.remove_linenums!(ex)

Remove all line-number metadata from expression-like object `ex`.
"""
function remove_linenums!(@nospecialize ex)
    ex isa Expr || return ex
```
This might be a good opportunity to enhance performance as well.

Reply:
Note there is still a separate method for `CodeInfo` as well.

Reply:
Ah, okay. Then it would be probably better to merge the method for `CodeInfo` into the generic one.

Reply:
Okay, let's see how that goes.